lib/components/settings/widgets/settings_card.dart

```diff
@@ -18,6 +18,7 @@ import 'package:flutter/material.dart';
 import 'dart:math' as math;
 import 'package:pangolin/components/settings/models/settings_element_model.dart';
 
+// ignore: must_be_immutable
```

Comment:
don't ignore this?? if you need it to be mutable use a class that is supposed to handle mutable items like stateful widgets